### PR TITLE
New/angular components/year range

### PR DESCRIPTION
**Pull request by yleaxeman**

### CODE REVIEW

File: projects/fhi-angular-components/src/lib/fhi-time/fhi-year-range/fhi-year-range.component.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  templateUrl: './fhi-year-range.component.html',
+  imports: [ FhiYearSelectorComponent ],
+  providers: [ FhiConstantsService ]
+})
+export class FhiYearRangeComponent {
+  @Input() labelFromYear: string = 'Fra år';
+  @Input() labelToYear: string = 'Til år';
+  @Input() maxYear: number = this.FHI_CONSTANTS.MAX_YEAR;
+  @Input() minYear: number = this.FHI_CONSTANTS.MIN_YEAR;
+
```
Comment (proand) [inline projects/fhi-angular-components/src/lib/fhi-time/fhi-year-range/fhi-year-range.component.ts:22]: No @output ?

File: projects/fhi-angular-components/src/lib/fhi-year-selector/fhi-year-selector.component.ts [hunk 2]
```diff
@@ -1,28 +1,41 @@
-import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Component, EventEmitter, Input, Output, SimpleChanges } from '@angular/core';
```
Comment (proand) [inline projects/fhi-angular-components/src/lib/fhi-year-selector/fhi-year-selector.component.ts:1]: SimpleChanges not in use...

File: projects/fhi-angular-components/src/lib/fhi-year-selector/fhi-year-selector.component.ts [hunk 3]
```diff
@@ -1,28 +1,41 @@
...
 import { FhiAutosuggestModule } from '../fhi-autosuggest/fhi-autosuggest.module';
 import { FhiAutosuggestItem } from '../fhi-autosuggest/fhi-autosuggest.model';
+
+import { FhiConstantsService } from '../shared-services/fhi-constants.service';
+
+import { getYear } from 'date-fns';
```
Comment (proand) [inline projects/fhi-angular-components/src/lib/fhi-year-selector/fhi-year-selector.component.ts:7]: getYear not in use...

File: projects/fhi-angular-components/src/lib/fhi-time/fhi-year-range/fhi-year-range.component.ts (hunk 1 above)
Comment (yleaxeman) [reply projects/fhi-angular-components/src/lib/fhi-time/fhi-year-range/fhi-year-range.component.ts:22]: I totally forgot... 😅

**Review by proand — APPROVED** [review]
Good enough for now.

**Pull request merged**